overlord/state/state.go

```diff
@@ -37,6 +38,29 @@ type Backend interface {
 	Checkpoint(data []byte) error
 }
 
+type customEntries map[string]*json.RawMessage
```

Comment:
I have a tempting feeling to name all of these as "data", "Data", and "customData", per the undefined nature of the contents. What do you think?